test/integration/change-streams/change_stream.test.ts

```diff
@@ -1679,19 +1171,19 @@ describe('Change Streams', function () {
           }
         });
         changeStream.stream().pipe(d);
-        waitForStarted(changeStream, () => {
-          collection.insertOne({ x: 1 }, (err, result) => {
-            expect(err).to.not.exist;
-            expect(result).to.exist;
-            triggerResumableError(changeStream, 250, () => {
-              collection.insertOne({ x: 2 }, (err, result) => {
-                expect(err).to.not.exist;
-                expect(result).to.exist;
-              });
-            });
-          });
-        });
-      })
+        // waitForStarted(changeStream, () => {
+        //   collection.insertOne({ x: 1 }, (err, result) => {
+        //     expect(err).to.not.exist;
+        //     expect(result).to.exist;
+        //     triggerResumableError(changeStream, 250, () => {
+        //       collection.insertOne({ x: 2 }, (err, result) => {
+        //         expect(err).to.not.exist;
+        //         expect(result).to.exist;
+        //       });
+        //     });
+        //   });
+        // });
+      }
```

Comment:
Did you mean to delete this?

Reply:
No, the test is skipped and connected to a ticket, but I didn't want to take the time to fix it since it is skipped, I have to comment this out because it's using functions that no longer exist.

Reply:
Ah I see now.  Makes sense!

Reply:
Going to leave this open in case Daria wonders too.